controllers/clustermanagementaddon_controller.go

```diff
@@ -152,43 +153,52 @@ func (c *ClusterManagementAddonReconciler) Reconcile(ctx context.Context, reques
 	}
 
 	// deploying central proxy server instances into the hub cluster.
-	err = c.deployProxyServer(config)
+	isRedeployed, err := c.deployProxyServer(config)
```

Comment:
maybe modified or deployed. isRedeployed is bit confusing

Reply:
renamed to isModified